nginx/ng/install.sls

```diff
@@ -51,7 +51,12 @@ nginx_zypp_repo:
 
 {% if salt['grains.get']('os_family') == 'RedHat' %}
 nginx_yum_repo:
-  pkgrepo.managed:
+  pkgrepo:
```

Comment:
I've done it the same way as it's done above in the suse section. But ofcourse i could change it to conform with style recomendation.